platform-api/src/main/java/io/datacater/core/stream/StreamEndpoint.java

```diff
@@ -80,20 +84,37 @@ public Uni<Response> createStream(Stream stream) throws JsonProcessingException
   @RequestBody
   @Consumes(MediaType.APPLICATION_JSON)
   public Uni<StreamEntity> updateStream(@PathParam("uuid") UUID uuid, Stream stream) {
+    Uni<List<ConfigEntity>> configList =
+        ConfigUtilities.getMappedConfigs(stream.configSelector(), dsf);
+
     return dsf.withTransaction(
-            ((session, transaction) ->
-                session
-                    .find(StreamEntity.class, uuid)
-                    .call(
-                        Unchecked.function(
-                            se -> {
-                              LOGGER.info(se);
-                              updateStreamObject(stream);
-                              return session.merge((se).updateEntity(stream));
-                            }))))
-        .onFailure()
-        .transform(
-            ex -> new UpdateStreamException(exceptionCauseMessageIfAvailable((Exception) ex)));
+        ((session, transaction) ->
+            session
+                .find(StreamEntity.class, uuid)
+                .onItem()
+                .transformToUni(
+                    streamEntity ->
+                        Uni.combine()
+                            .all()
+                            .unis(Uni.createFrom().item(streamEntity), configList)
+                            .asTuple())
+                .onItem()
+                .transform(
+                    tuple -> {
+                      LOGGER.info(tuple.getItem1());
```

Comment:
Can you please add some more information/context to the log message, which helps us to understand what you are logging here? Assume that we see this log message among many other log messages.